ozpricechecker/pricefinderapp/models.py

```diff
@@ -1,3 +1,115 @@
-# from django.db import models
+from django.db import models
+from django.contrib.auth.models import User
+
 
 # Create your models here.
+class Currency(models.Model):
+    name = models.CharField(max_length=3, unique=True)
+
+    def __str__(self):
+        return self.name
+
+
+class Store(models.Model):
+    name = models.CharField(max_length=30, unique=True)
+    prod_base_url = models.TextField(unique=True)
+    dynamic_page = models.BooleanField(default=False)
+    currency = models.ForeignKey(Currency, on_delete=models.CASCADE)
+
+    def __str__(self):
+        return self.name
+
+
+class Product(models.Model):
+    store = models.ForeignKey(Store, on_delete=models.CASCADE)
+    prod_url = models.TextField(unique=True)
+    name = models.CharField(max_length=30)
+
+    def __str__(self):
+        return self.name
+
+
+class ProductPrice(models.Model):
+    product = models.ForeignKey(Product, on_delete=models.CASCADE)
+    date_time = models.DateTimeField(auto_now=True)
+    price = models.DecimalField(max_digits=5, decimal_places=2)
```

Comment:
What's the max Value? 999.99 or 99999.99.
Do we need this limitation of price?
We probably need to decide whether we capture the price in e.g. cents only or dollar.cents.